discussions/discussion09/discussion09-LinR.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>Make sure you define homescedasticity because this wasn't defined in class.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/ds701_fa2024_assignments/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>